packages/ember-data/tests/integration/adapter/find-test.js

```diff
@@ -9,6 +9,9 @@ module("integration/adapter/find - Finding Records", {
       firstName: DS.attr('string'),
       lastName: DS.attr('string')
     });
+    Person.typeKey = 'person';
+    Person.modelName = 'person';
+
```

Comment:
Since #3033 is merged and `references` is rebased onto `master`, this can be removed.